### PR TITLE
WRO-11096: Added support for running performance tests on TV

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added support for running performance tests on TV:
- package.json: updated enact and react dependencies; added new script for running test on TV. The targetEnv is specified each script
- jest.config: increased timeout to 50000 because tests are slower on TV board.
- puppeteer.setup.js:  targetEnv parameter and TV_IP environment variable are read. For each target environment, different actions are performed before and after each test; The server address is also read dinamically, in order to add support for remote addresses
- test files: each tests was adapted to run on both PC and TV

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-11096

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

File: puppeteer.setup.js [hunk 1]
```diff
@@ -13,26 +14,61 @@ global.maxLCP = 2500;
...
+	global.afterEach(async () => {
+		await page.close();
+	});
+
+	global.afterAll(async () => {
+		await browser.close();
+	});
+} else if (targetEnv === 'TV') {
+	global.beforeAll(async () => {
+		const TVAddr = process.env.TV_IP; // "10.255.248.96:9998";
```
Comment (jeonghee27) [inline puppeteer.setup.js:48]:
```suggestion
		const TVAddr = process.env.TV_IP;
```
Please remove actual IP.

File: puppeteer.setup.js [hunk 2]
```diff
@@ -13,26 +14,61 @@ global.maxLCP = 2500;
...
+
+		browser = await puppeteer.connect({
+			browserURL: `http://${TVAddr}:9998`,
+			ignoreHTTPSErrors: true
+		});
+
+		const pages = await browser.pages();
+		// const pagesCount = pages.length;
```
Comment (jeonghee27) [inline puppeteer.setup.js:56]:
```suggestion
```

File: puppeteer.setup.js [hunk 3]
```diff
@@ -13,26 +14,61 @@ global.maxLCP = 2500;
...
+
+		global.testMultiple = browser;
+		global.testMultipleTV = browser;
```
Comment (jeonghee27) [inline puppeteer.setup.js:59]: Do we need this 'testMultipleTV' variable?

**Review by jeonghee27 — APPROVED** [review]
LGTM

**Pull request merged**